frontend/src/components/FormEndPage/StatusTrackerLink.tsx

```diff
@@ -0,0 +1,68 @@
+import { BiLinkExternal } from 'react-icons/bi'
+import {
+  Box,
+  FormControl,
+  InputGroup,
+  InputRightElement,
+  Skeleton,
+  Stack,
+} from '@chakra-ui/react'
+
+import FormLabel from '~components/FormControl/FormLabel'
+import IconButton from '~components/IconButton'
+import Input from '~components/Input'
+import { CopyButton } from '~templates/CopyButton'
+
+type StatusTrackerLinkProps = {
+  formId: string | undefined
```

Comment:
Doesn't look like the callers will provide empty `formId`
```suggestion
  formId: string
```